app/uk/gov/hmrc/apidocumentation/models/Documentation.scala

```diff
@@ -46,6 +54,18 @@ case class APIDefinition(
   lazy val statusSortedActiveVersions = statusSortedVersions.filterNot(v => v.status == APIStatus.RETIRED)
   lazy val defaultVersion = statusSortedActiveVersions.headOption
   lazy val hasActiveVersions = statusSortedActiveVersions.nonEmpty
+  lazy val label: APIDefinitionLabel = if (isTestSupport.getOrElse(false)) {
+    TEST_SUPPORT_API
+  } else if (isXmlApi.getOrElse(false)) {
+    XML_API
+  } else {
+    REST_API
+  }
+
+  def mappedCategories(catMap: Map[String, Seq[APICategory]] = categoryMap): Seq[APICategory] = categories match {
+    case Some(head :: tail) => head +: tail
```

Comment:
I think this equivalent to Some(x) => x.

Reply:
`Some(x) => x` would match `Some(Nil)`, whereas this is looking for a non-empty list (to cope with `categories: []` in the definition, or the default of `Nil`):

```scala
val categories: Option[Seq[String]] = Some(Nil)

categories match {
  case Some(x) => x
  case _ => Seq("default")
}

categories: Option[Seq[String]] = Some(List())
res2: Seq[String] = List()
```
vs
```scala
val categories: Option[Seq[String]] = Some(Nil)

categories match {
  case Some(head :: tail) => head +: tail
  case _ => Seq("default")
}

categories: Option[Seq[String]] = Some(List())
res3: Seq[Any] = List(default)
```